src/index.js

```diff
@@ -229,7 +355,34 @@ class ClientAuthOIDC {
       })
   }
 
-  clearCurrentUser () {
+  /**
+   * Saves given user credentials in storage.
+   *
+   * @param {Object} options
+   */
+  saveCurrentCredentials (options) {
+    this.setCurrentCredentials(options)
+    this.store.setItem(CURRENT_CREDENTIALS, JSON.stringify(options))
+  }
+
+  /**
+   * Initializes user credentials on the client instance.
+   *
+   * @param {Object} options
+   */
+  setCurrentCredentials (options) {
+    this.webId = options.webId
+    this.accessToken = options.accessToken
+    this.idToken = options.idToken
+  }
```

Comment:
I see what you're saying, re sources of truth. I'll change it to just rely on storage.